canvas_course_admin_tools/settings/local.py

```diff
@@ -42,6 +42,9 @@
     'icommons_rest_api': {
         'base_path': 'api/course/v2'
     },
+    'manage_course': {
+        'relative_url': 'courses/27/external_tools/170',  # dev (Manage Course)
```

Comment:
Does this mean we can remove the URLs for `manage_people`, and that we need to move the comment above those URLs up here?